GitUI/CommandsDialogs/BrowseDialog/FormGitCommandLog.Designer.cs

```diff
@@ -30,263 +30,273 @@ protected override void Dispose(bool disposing)
         /// </summary>
         private void InitializeComponent()
         {
-            this.components = new System.ComponentModel.Container();
```

Comment:
Should we make this change globally?

Reply:
I thought yes. But this will require much testing and effortful review. F.i. FormBrowse is broken on save.

Reply:
> Could you try [#11186 (comment)](https://github.com/gitextensions/gitextensions/pull/11186#issuecomment-1711090037)?

The designer ignores this setting on save. So I would continue to update forms when we touch them. Then we can crosscheck that it has not got broken.

Reply:
@KlausLoeffelmann FYI

Reply:
Ah, damn, my bad. The change is for 17.8 (I think it went in in Preview 1 or 2). We have not serviced it back. Yet. Also, there will be minor differences in code generation from 17.5.3 on, one way or the other. The setting should be honered, though, from 17.8 on going forward. If you continue to see issues/major regressions in code gen, can you guys open an issue in the WinForms runtime (or @RussKie in the Designer) repo? Thanks!

Reply:
Thank you for the clarification. This is not such urgent and not a regression.
The specialties with FormBrowse are caused by manual modifications.